api/views.py

```diff
@@ -740,10 +742,13 @@ def post(self, request, format=None):
                 user.profile.is_referred = True
                 user.profile.referred_by = queryset[0]
 
+            user.profile.wants_stealth = False
+
             user.profile.save()
 
             context["public_key"] = user.profile.public_key
             context["encrypted_private_key"] = user.profile.encrypted_private_key
+            context["wants_stealth"] = False
```

Comment:
Looks good, but if we remove the line above it would make sense to change this one to
```python

context["wants_stealth"] = user.profile.wants_stealth
```